project/GenerateAnyVals.scala

```diff
@@ -273,13 +273,16 @@ import scala.language.implicitConversions"""
 }
 
 trait GenerateAnyValTemplates {
-  def headerTemplate = """/*                     __                                               *\
-**     ________ ___   / /  ___     Scala API                            **
-**    / __/ __// _ | / /  / _ |    (c) 2002-2013, LAMP/EPFL             **
-**  __\ \/ /__/ __ |/ /__/ __ |    http://scala-lang.org/               **
-** /____/\___/_/ |_/____/_/ | |                                         **
-**                          |/                                          **
-\*                                                                      */
```

Comment:
RIP ASCII art.

Reply:
at the risk of bikeshedding (but, we only want to do this once):

the main reason I was interested in seeing the ASCII art die was to save vertical screen real estate, it wasn't actually about the ASCII art itself.

wdyt about condensing

```
/*
 * Scala
 * (c) EPFL
 * (c) Lightbend, Inc.
 * https://www.scala-lang.org
 * Licensed under Apache License 2.0
 *
 * See the NOTICE file distributed with this work for additional information
 * regarding copyright ownership.
 */
```

to

```
/* Scala. (c) EPFL, (c) Lightbend, Inc. Apache License 2.0.
    See NOTICE file for full license and copyright info. */
```

Reply:
Btw, the "standard license header" is quite long: https://spdx.org/licenses/Apache-2.0.html#licenseHeader

Reply:
I am personally meh about saving some vertical lines on top of the file, but I don't work with this code base every day. `/*` at the top and bottom is generated by sbt-header, so it's out my control. If you could tally up the Scala team's opinion on exact header on top, I can make another change. There's adjustment of patch file I have to do etc, so it's not super easy.

Reply:
Personally, I don't mind the vertical lines at the top of the file. It's not that long, and honestly feels... like it makes the copyright/license notice less ugly and annoying? idk. that's just how I personally feel.

Also I feel like the ASCII art makes it easy to tell at a glace that it's the copyright header, whereas with the suggested 2 line version, it's less obvious and (without reading) might be a comment about the internal structure of the file or something.

Reply:
Let's go with the following semi-compromise (the usual outcome of bikeshedding...):

```
Scala (${(ThisBuild/homepage).value.get})

Copyright EPFL and Lightbend, Inc.

Licensed under the Apache License, Version 2.0
(http://www.apache.org/licenses/LICENSE-2.0).

See the NOTICE file distributed with this work for 
additional information regarding copyright ownership.
```

Reply:
The main thing I care about: spell out "Copyright", leave off the year so we don't need to bother updating it. If this ever goes to court, there's plenty of evidence on the blockchain^W git.

Reply:
Done https://github.com/scala/scala/blob/2d9e6acce8c4246e6cba5a22ff9d965ec3bd117c/src/library/scala/Array.scala

Reply:
Thank you! Let's merge this before we run into merge conflicts @adriaanm

Reply:
To counter the encroaching Swiss minimalist contingency I am sending a PR to add ASCII art to sbt/sbt's startup - https://github.com/sbt/sbt/pull/4402/commits/0d788162aef45eb6efea7713208c508074823e4f

Reply:
What a shock to update! It's still broken on 2.13, but now I recognize the historical importance of:
```
$ scala -Dscala.banner=welcome

     ________ ___   / /  ___  
    / __/ __// _ | / /  / _ | 
  __\ \/ /__/ __ |/ /__/ __ | 
 /____/\___/_/ |_/____/_/ | | 
                          |/  version 2.12.6

scala> 
```

Reply:
Whew...
```
$ scalam -Dscala.repl.welcome=banner

     ________ ___   / /  ___
    / __/ __// _ | / /  / _ |
  __\ \/ /__/ __ |/ /__/ __ |
 /____/\___/_/ |_/____/_/ | |
                          |/  version 2.13.0-M5

scala> 
```
That should go in the release notes...